source/configuring_domain/05_configuring_domain/tabs.rst

```diff
@@ -27,3 +27,28 @@ See :doc:`Visualizing data </exploring_catalog_and_datasets/02_looking_up_a_data
 By default the dataset is opened on the **table view**, but you can choose to display the **information tab** instead. The table view by default is useful to give users with a direct access to the data, whereas the information tab by default is interesting to provide users with context and description first.
 
 .. image:: images/configuration_default-tab.png
+
+
+Catalog page — Geographic filter
+--------------------------------
+
+The geographic filter allows to search datasets in the catalog based on the geographical location of their data (see :ref:`Filtering the catalog using the geographic filter <using-the-minimap>`). By default, this filter is not activated.
+
+.. admonition:: Important
+   :class: important
+
+   The geographic filter can either enable filtering at international level or only on a specific, chosen country.
+
```

Comment:
Now that we don't have a mode named "International" anymore, maybe using a more explicit term would be better. For example "filtering across the entire world, or only within a specific, chosen country"?